mlir/lib/Dialect/Linalg/Transforms/DropUnitDims.cpp

```diff
@@ -252,7 +253,7 @@ replaceUnitDimIndexOps(GenericOp genericOp,
 /// Expand the given `value` so that the type matches the type of `origDest`.
 /// The `reassociation` is used when `rankReductionStrategy` is set to
 /// `RankReductionStrategy::ReassociativeReshape`.
-static Value
+static FailureOr<Value>
```

Comment:
This is not returning a failure yet? So then this doesnt need to be a `FailureOr`